Assignment/solution_student/aboutme.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html> 
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>About Me</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+	<nav class="nav text-center">
+		<a href="aboutme.html" class="nav-link nav-link-active">About Me</a> |
+		<a href="portfolio.html" class="nav-link">Portfolio</a>
+	</nav>
+
+	<h1 class="text-center">Joe Johnson</h1>
+
+	<hr>
+
+	<img src="images/avatar.png" alt="Avatar" align="right" class="avatar image-border">
```

Comment:
Very clever, but remember, we don't want to put _any_ styling in our markup, so doing `align="right"` is not an acceptable method —instead achieve the same effect by applying `float: right;` to the `.avatar` class in your CSS :)